contrib/bloop/test/src/mill/contrib/bloop/BloopTests.scala

```diff
@@ -25,7 +25,7 @@ object BloopTests extends TestSuite {
 
     object scalaModule extends scalalib.ScalaModule with testBloop.Module {
       def scalaVersion = "2.12.8"
-      val bloopVersion = "1.4.6"
+      val bloopVersion = "1.4.6-33-1c6f6712"
```

Comment:
If the intention is, to use the same version for testing as for building, then we should probably think about generating some `Versions` file in `build.sc` and use it here, as we do in other modules.

Reply:
✅  Done!

Reply:
Nice. I'm fine to merge after CI succeedes. Your PRs are a pleasure.